### PR TITLE
[chassis][voq]Fix the test test_techsupport.py for voq linecards

### LINKED ISSUE #7510: test_techsupport_commands fails on DNX based platforms due to incorrect bcmcmd specfications
test_techsupport_commands fail with an error because the test is not aware of DNX family and tries to execute common commands.

**Steps to reproduce the issue:**

run the above test on any DNX platform.

**Describe the results you received:**

```
--- FAILED: test_techsupport_commands[cmp227-4] Failed!
Failed: defaultdict(<type 'list'>, {'broadcom_cmd_bcmcmd': ['bcmcmd -n 0 "l3 nat_ingress show"', 'bcmcmd -n 1 "l3 nat_ingress show"', 'bcmcmd -n 0 "l3 nat_egress show"', '\
bcmcmd -n 1 "l3 nat_egress show"', 'bcmcmd -n 0 "ipmc table show"', 'bcmcmd -n 1 "ipmc table show"', 'bcmcmd -n 0 "multicast show"', 'bcmcmd -n 1 "multicast show"', 'bcmcm\
d -n 0 "fp show"', 'bcmcmd -n 1 "fp show"', 'bcmcmd -n 0 "pvlan show"', 'bcmcmd -n 1 "pvlan show"', 'bcmcmd -n 0 "l3 intf show"', 'bcmcmd -n 1 "l3 intf show"', 'bcmcmd -n
```

**Describe the results you expected:**

The test should have passed.

The fix is to have checks in the test to expect the specific tech_support commands based on family.

### PR DESCRIPTION
**Pull request by arlakshm**
<!--
Please make sure you've read and understood our contributing guidelines;
https://github.com/sonic-net/SONiC/blob/gh-pages/CONTRIBUTING.md

Please provide following information to help code review process a bit easier:
-->
### Description of PR
<!--
- Please include a summary of the change and which issue is fixed.
- Please also include relevant motivation and context. Where should reviewer start? background context?
- List any dependencies that are required for this change.
-->

Summary:
Fixes #7510 

### Type of change

<!--
- Fill x for your type of change.
- e.g.
- [x] Bug fix
-->

- [x ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)


### Back port request
- [ ] 201911
- [ ] 202012
- [ ] 202205

### Approach
#### What is the motivation for this PR?
PR https://github.com/sonic-net/sonic-mgmt/pull/7998 has partial fixes. Additional changes are required which is added in this PR

#### How did you do it?
fix the brcm dnx commands in tech support commands.
Fix docker commands

#### How did you verify/test it?
Run `show_techsupport/test_techsupport.py::test_techsupport_command`
```
arlakshm@d0cac380f2f2:/data/my_mgmt/tests$ sudo ANSIBLE_KEEP_REMOTE_FILES=1 ./run_tests.sh -c show_techsupport/test_techsupport.py::test_techsupport_commands[str2-sonic-lc5-1] -i '../ansible/str2,../ansible/veos' -n vms26-t2-7800-1 -t 't2,any' -e "--skip_sanity --pdb" -u
=== Running tests in groups ===
Running: pytest show_techsupport/test_techsupport.py::test_techsupport_commands[str2-sonic-lc5-1] --inventory ../ansible/str2,../ansible/veos --host-pattern str2-sonic-lc5-1,str2-sonic-lc3-1,str2-sonic-lc7-1,str2-sonic-sup-1 --testbed vms26-t2-7800-1 --testbed_file /data/my_mgmt/ansible/testbed.yaml --log-cli-level warning --log-file-level debug --kube_master unset --showlocals --assert plain --show-capture no -rav --allow_recover --ignore=ptftests --ignore=acstests --ignore=saitests --ignore=scripts --ignore=k8s --ignore=sai_qualify --junit-xml=logs/tr.xml --log-file=logs/test.log --topology t2,any --skip_sanity --pdb
/usr/local/lib/python2.7/dist-packages/ansible/parsing/vault/__init__.py:44: CryptographyDeprecationWarning: Python 2 is no longer supported by the Python core team. Support for it is now deprecated in cryptography, and will be removed in the next release.
  from cryptography.exceptions import InvalidSignature
===================================================================================================================================================== test session starts ======================================================================================================================================================
platform linux2 -- Python 2.7.18, pytest-4.6.11, py-1.11.0, pluggy-0.13.1
ansible: 2.8.12
rootdir: /data/my_mgmt/tests, inifile: pytest.ini
plugins: forked-1.3.0, metadata-1.11.0, ansible-2.2.4, celery-4.4.7, xdist-1.28.0, html-1.22.1, allure-pytest-2.8.22, repeat-0.9.1
collecting ... /usr/local/lib/python2.7/dist-packages/ansible/parsing/vault/__init__.py:44: CryptographyDeprecationWarning: Python 2 is no longer supported by the Python core team. Support for it is now deprecated in cryptography, and will be removed in the next release.
  from cryptography.exceptions import InvalidSignature
/usr/local/lib/python2.7/dist-packages/ansible/parsing/vault/__init__.py:44: CryptographyDeprecationWarning: Python 2 is no longer supported by the Python core team. Support for it is now deprecated in cryptography, and will be removed in the next release.
  from cryptography.exceptions import InvalidSignature
/usr/local/lib/python2.7/dist-packages/ansible/parsing/vault/__init__.py:44: CryptographyDeprecationWarning: Python 2 is no longer supported by the Python core team. Support for it is now deprecated in cryptography, and will be removed in the next release.
  from cryptography.exceptions import InvalidSignature
/usr/local/lib/python2.7/dist-packages/ansible/parsing/vault/__init__.py:44: CryptographyDeprecationWarning: Python 2 is no longer supported by the Python core team. Support for it is now deprecated in cryptography, and will be removed in the next release.
  from cryptography.exceptions import InvalidSignature
/usr/local/lib/python2.7/dist-packages/ansible/parsing/vault/__init__.py:44: CryptographyDeprecationWarning: Python 2 is no longer supported by the Python core team. Support for it is now deprecated in cryptography, and will be removed in the next release.
  from cryptography.exceptions import InvalidSignature
collected 1 item                                                                                                                                                                                                                                                                            

show_techsupport/test_techsupport.py::test_techsupport_commands[str2-sonic-lc5-1] PASSED                                                                                                                                                                                                                                  [100%]
```

#### Any platform specific information?

#### Supported testbed topology if it's a new test case?

### Documentation
<!--
(If it's a new feature, new test case)
Did you update documentation/Wiki relevant to your implementation?
Link to the wiki page?
-->

### CODE REVIEW

**Comment by kartik-arista** [comment]
Looks good @arlakshm.

**Comment by kenneth-arista** [comment]
Looks good to me. I don't have the permission to approve. Hopefully someone else can do the needful.

File: tests/show_techsupport/tech_support_cmds.py [hunk 1]
```diff
@@ -212,73 +221,72 @@
...
+    'bcmcmd{} "tm ing q map"',
+    'bcmcmd{} "tm ing vsq resources"',
+    'bcmcmd{} "tm ing vsq non g=a"',
+    'bcmcmd{} "tm ing vsq non g=b"',
+    'bcmcmd{} "tm ing vsq non g=c"',
+    'bcmcmd{} "tm ing vsq non g=d"',
+    'bcmcmd{} "tm ing vsq non g=e"',
+    'bcmcmd{} "tm ing vsq non g=f"',
+    'bcmcmd{} "fabric connectivity"',
+    'bcmcmd{} "port status"',
```
Comment (rlhui) [inline tests/show_techsupport/tech_support_cmds.py:289]: Have we tested and confirmed with Brcm that these commands are all good to use in production?

Comment (kenneth-arista) [reply tests/show_techsupport/tech_support_cmds.py:289]: We've tested this PR and also raised CS00012296351 with Brcm as requested.

**Review by rlhui — APPROVED** [review]

**Pull request merged**